### DISCUSSION TITLE
Popup on opening links in a new tab

### DISCUSSION THREAD

**Discussion by spcnko** [zen-browser/desktop#1007] [Ideas]
It would be nice to see a popup after opening a link in a new tab from the website you are browsing (using middle click on a link for example). Some times it's hard to know if I opened a link, or if it failed, or I missclicked because there is no feedback. For context I use compact mode so I don't see the panel with tabs until I open it to check. 

An example of this could be how Arc Browser does it where you see a little popup on the top right corner of the browser saying you opened a link in a new tab.

**Comment by ahmaadaziz** [comment]
Yes, this would be greatly appreciated. I have also felt the need for some feedback on opening a new tab or even when reloading a page with F5 or CTRL + R.

**Comment by 0xAdi0x** [comment]
+1
